pkg/sandboxes/aws-ecs-byovpc/iam.go

```diff
@@ -0,0 +1,42 @@
+package awsecsbyovpc
+
+import perms "github.com/nuonco/sandboxes/pkg/sandboxes/permissions"
+
+var ProvisionPermissions = append([]string{
+	"ecs:CreateCapacityProvider",
+	"ecs:CreateCluster",
+	"ecs:DescribeCapacityProviders",
+	"ecs:DescribeClusters",
+	"ecs:ListTagsForResource",
+	"ecs:PutClusterCapacityProviders",
+	"ecs:TagResource",
+}, perms.BaseProvisionPermissions...)
+
+var ProvisionPolicy = perms.Policy{
+	Version: "2012-10-17",
+	Statement: []perms.Statement{
+		{
+			Sid:      "",
+			Effect:   "Allow",
+			Resource: "*",
+			Action:   ProvisionPermissions,
+		},
+	},
+}
+
+var DeprovisionPermissions = append([]string{
+	"ecs:DeleteCapacityProvider",
+	"ecs:DeleteCluster",
+}, perms.BaseDeprovisionPermissions...)
+
+var DeprovisionPolicy = perms.Policy{
+	Version: "2012-10-17",
+	Statement: []perms.Statement{
+		{
+			Sid:      "",
```

Comment:
Empty string is the zero value for string types - no need to initialize it this way

```suggestion
```